README.md

```diff
@@ -69,6 +69,7 @@ One of the easiest ways to contribute is to participate in discussions. There ar
 |---------------------|---|
 | Mailing List | https://groups.google.com/forum/#!forum/oam-dev |
 | Community meeting info | [Bi-weekly (Starting Oct 22, 2019), Tuesdays 10:30AM PST](https://calendar.google.com/calendar?cid=dDk5YThyNGIwOWJyYTJxajNlbWI0a2FvdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)  |
+| APAC Friendly Community meeting | [Bi-weekly (Starting Oct 28, 2019), Mondays 5:00PM PST](https://calendar.google.com/calendar?cid=dDk5YThyNGIwOWJyYTJxajNlbWI0a2FvdGtAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)  |
```

Comment:
Emm, the calendar link is same?

Also, I noticed the APAC community meeting link in https://github.com/oam-dev/rudr#contributing seems not working ("not found")?

Reply:
The calendar link is the same because the calendar contains all the events. 

- Let me take a look and fix that other link.

Reply:
This is fine. I think Mondays 4:00PM PST is also OK, if it's possible for matt to join.